frontend/src/__tests__/setupTests.js

```diff
@@ -0,0 +1,15 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
+
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+// dummy test to make jest not complain about no tests
+test('renders learn react link', () => {
```

Comment:
Was this verified to be compatible with the newest Redux changes in #133 ?

Reply:
I've merged main into this branch yesterday, so it should work.